### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by UlybinSergej**

### CODE REVIEW

File: src/task6/MyArrayList.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package task6;
+
+import java.util.Arrays;
+
+public class MyArrayList<T> {
+    private Object[] list;
+    private int length;
```
Comment (YanBelikov) [inline src/task6/MyArrayList.java]: I think it should be static

File: src/task6/BicyclesModel.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    MOUNTAIN_BIKE(1),
+    CITY_BIKE(2),
+    TRIAL_BIKE(3);
+
+    private int bikeNumber;
+
+    BicyclesModel(int bikeNumber) {
+        this.bikeNumber = bikeNumber;
+    }
+}
```
Comment (YanBelikov) [inline src/task6/BicyclesModel.java]: you can put it into class Bycicle

File: src/task6/Bicycle.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            throw new MaxSpeedException("Максимальная скорость вне допустимого диапазона!(5 <= maxSpeed <= 50)");
+        }
+    }
+
+    @Override
+    public String toString() {
+        return "Bicycle{" +
+                "modelName=" + modelName +
+                ", maxSpeed=" + maxSpeed +
+                '}';
```
Comment (YanBelikov) [inline src/task6/Bicycle.java]: merge in one line

Comment (UlybinSergej) [reply src/task6/Bicycle.java]: Done

File: src/task6/BicyclesModel.java (hunk 2 above)
Comment (UlybinSergej) [reply src/task6/BicyclesModel.java]: Done

File: src/task6/MyArrayList.java (hunk 1 above)
Comment (UlybinSergej) [reply src/task6/MyArrayList.java]: But if variable "length" would be static - then all objects of type MyArrayList would be same variable "length"

**Review by YanBelikov — APPROVED** [review]

**Pull request merged**